pkg/networkservice/chains/nsmgr/suite_test.go

```diff
@@ -795,6 +796,7 @@ func additionalFunctionalityChain(ctx context.Context, clientURL *url.URL, clien
 					client.WithAdditionalFunctionality(
 						mechanismtranslation.NewClient(),
 						replacelabels.NewClient(labels),
+						replacensename.NewClient(),
```

Comment:
Im mostly sure that we could face with the similar issue in future (when we need to replace something in pass through requests)

Should we create a pkg like `passthrough` and concatenate all needed  replace elements?

Something like

```go
package passthrough

func NewClient(labels map[string]string) {
return chain.NewClient(
replacelabels.NewClient(labels),
replacensename.NewClient(),
)

}
``` 

And use `passthrough` chain in firewall nses?


/cc @edwarnicke

Reply:
It's a good idea, thanks! 
Done